Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -267,15 +267,7 @@ class FontCascade : public CanMakeWeakPtr<FontCascade> {
 
     FontSelector* fontSelector() const;
     static bool treatAsSpace(UChar32 c) { return c == space || c == tabCharacter || c == newlineCharacter || c == noBreakSpace; }
-    static bool isCharacterWhoseGlyphsShouldBeDeletedForTextRendering(UChar32 character)
```

Comment:
Is it really OK to un-inline this function?

Reply:
I made it inline again.